playbooks/templates/slack_notification_on_alert.json

```diff
@@ -0,0 +1,218 @@
+        "name": "Get Events",
+        "type": "action",
+        "outputs": {
+          "default": [
+            "10"
+          ]
+        },
+        "arguments": {
+          "query": "event.id:\"{{ node.5.default.value.split('--').1}}\"",
+          "latest_time": "{{ node.3['last_seen_at'] }}",
+          "earliest_time": "{{ node.3['first_seen_at'] }}"
+        },
+        "action_uuid": "af0b4355-a428-43d6-991c-d5ff878e17d5",
+        "module_uuid": "92d8bb47-7c51-445d-81de-ae04edbb6f0a"
+      },
+      "7": {
+        "name": "Store",
+        "type": "operator",
+        "outputs": {
+          "default": []
+        },
+        "subtype": "store",
+        "modifications": [
+          {
+            "key": "targets",
+            "type": "append",
+            "value": "{{ node.6['events'][0]['host.name'] }}"
+          }
+        ]
+      },
+      "8": {
+        "name": "Condition",
+        "type": "operator",
+        "cases": [
+          {
+            "left": "{% if node.3['target'] is not none %}true1{% endif %}",
+            "name": "target defined",
+            "right": "true1",
+            "comparison": "=="
+          }
+        ],
+        "outputs": {
+          "else": [
+            "4"
+          ],
+          "target defined": [
+            "9"
+          ]
+        },
+        "subtype": "condition"
+      },
+      "9": {
+        "name": "Store",
+        "type": "operator",
+        "outputs": {
+          "default": [
+            "2"
+          ]
+        },
+        "subtype": "store",
+        "modifications": [
+          {
+            "key": "targets",
+            "type": "append",
+            "value": "{{ node.3.target }}"
+          }
+        ]
+      },
+      "10": {
+        "name": "Condition",
+        "type": "operator",
+        "cases": [
+          {
+            "left": "{% if node.6['events'][0]['host.name'] is defined%}true1{% endif %}",
+            "name": "host exists",
+            "right": "true1",
+            "comparison": "=="
+          }
+        ],
+        "outputs": {
+          "else": [],
+          "host exists": [
+            "7"
+          ]
+        },
+        "subtype": "condition"
+      }
+    },
+    "workspace": "Operation Center",
+    "description": "Push notification to slack"
```

Comment:
```suggestion
    "description": "Notify by Slack when new alerts are created. If the target is defined, it is used in the notification, otherwise the hostnames from events are used."
```

Reply:
Thank you very much for your contribution @sl-teddyferdinand  ! I just suggested to change a bit the name and the description and I added tags to allow filtering in the template list.

Reply:
good idea for the tag, I like that! :)